<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-FKHOH3U" failures="1" tests="2" name="Default test(failed)" time="25.039" errors="0" timestamp="2023-01-05T14:23:22 IST">
  <testcase classname="FrameworkKeywords.FailedTestCase" name="Test3" time="0.08">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at FrameworkKeywords.FailedTestCase.Test3(FailedTestCase.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- Test3 -->
  <testcase classname="FrameworkKeywords.FailedTestCase" name="Test2" time="0.052"/>
  <testcase classname="FrameworkKeywords.FailedTestCase" name="Test4" time="0.0">
    <ignored/>
  </testcase> <!-- Test4 -->
  <testcase classname="FrameworkKeywords.FailedTestCase" name="Test5" time="0.0">
    <ignored/>
  </testcase> <!-- Test5 -->
  <testcase classname="FrameworkKeywords.FailedTestCase" name="Test1" time="0.0">
    <ignored/>
  </testcase> <!-- Test1 -->
</testsuite> <!-- Default test(failed) -->
